openlane_pr_tag/action.yml

```diff
@@ -0,0 +1,40 @@
+name: Create a new pull request to update OpenLane tag.
+
+inputs:
+  ol_tag:
+    description: OpenLane tag to use in PR
+    required: true
+  path:
+    description: Repo path
+    required: true
+  ol_tag_file:
+    description: File where to store the new OpenLane tag
+    default: .github/openlane-stable-tag
+
+runs:
+  using: composite
+
+  steps:
+
+    - name: Update stable tag
+      shell: bash
+      run: |
+        cd ${{ inputs.path }}
+        echo "${{ inputs.ol_tag }}" > ${{ inputs.ol_tag_file }}
+
+    - name: Export tag name
+      run: echo "OL_TAG=${{ inputs.ol_tag }}" >> $GITHUB_ENV
```

Comment:
is the only reason we're exporting here is to be able to reuse it in the commit message?

Reply:
Yes, before adb37c90a203c5ae214bbd45a0f33172a33b9928 the input was not being correctly resolved.